queries/t32/indents.scm

```diff
@@ -0,0 +1,36 @@
+; Subroutines & scoping
+(block
+  ")" @indent.branch .) @indent.begin
+
+
+; Control flow statements
+(
+  (if_block
+    _*
+    condition: (_)
+    . (_) @_then) @indent.begin
+  (#not-has-type? @_then block)
+)
+
+(else_block) @indent.branch
+
+(else_block
+  (if_block) @indent.dedent) @indent.branch
+
+(
+  (else_block
+    (if_block
+      _*
+      condition: (_)
+      . (_) @_then)) @indent.branch
+  (#not-has-type? @_then block)
+)
+
+(while_block
+  (command_expression) @indent.end .) @indent.begin
+
+(repeat_block
+  (command_expression) @indent.end .) @indent.begin
```

Comment:
I see `(command_expression)` here too, can you give me an example of what this node is?

Reply:
This is the most basic case, too:
```
RePeaT &a
  print
```
A `command_expression` like `Data.Set VM:0x0--0xFF 0xA` starts with some command-dot-subcommands sequence and then continues with space- or comma-separated arguments. This command is for filling the memory range from `0` to `0xFF` with the value `0xA`.
There are many commands, so the possible arguments permutations are arbitrary. Commands terminate with the end of the line, so it is impossible to check for a special character sequence.